pkg/kvdpa/mgmtdev.go

```diff
@@ -0,0 +1,114 @@
+package kvdpa
+
+import (
+	"strings"
+	"syscall"
+
+	"github.com/vishvananda/netlink/nl"
+	"golang.org/x/sys/unix"
+)
+
+// MgmtDev represents a Vdpa Management Device
+type MgmtDev interface {
+	BusName() string // Optional
+	DevName() string //
+	Name() string    // The MgmtDevName is BusName/DevName
+	// TODO: SupportedClasses() []string
```

Comment:
What is the purpose of "SupportedClasses"? Is it still a TODO?

Reply:
It's one of the attributes of the Management Device: https://elixir.bootlin.com/linux/latest/source/include/uapi/linux/vdpa.h#L33
I commented it out because I had not added support for it yet.
But it's true it should not be there. I'll remove it.